.pre-commit-config.yaml

```diff
@@ -1,7 +1,9 @@
 ---
+default_language_version:
+  python: python3.11
```

Comment:
I'm not sure why you set this, but doing so prevents `pre-commit` from running successfully when Python 3.11 isn't present. That's annoying when running `pre-commit` locally, so please remove it again.

Reply:
That was an earlier attempt at fixing `prospector`. I had a strange error that seemed to indicate a breakage due a recent version of Python